webpack.static-viz.config.js

```diff
@@ -59,5 +61,29 @@ module.exports = env => {
     optimization: {
       minimize: !shouldDisableMinimization,
     },
+    plugins: [
+      new StatsWriterPlugin({
+        stats: {
+          modules: true,
+          assets: false,
+          nestedModules: false,
+          reasons: false,
+          excludeModules: [/node_modules/],
+        },
+        filename: "../../../../.github/static-viz-sources.yaml",
```

Comment:
nit: can we use some relative to project path here?

Reply:
I don't actually think we can. The path needs to be relative to `output.path` :/